### DISCUSSION TITLE
Does QGIS and SCP Work on a Mac

### DISCUSSION THREAD

**Discussion by anandpethia** [semiautomaticgit/SemiAutomaticClassificationPlugin#259] [Q&A]
Kindly advise if QGIS and SCP work on a Mac. Have been jumping from one error message to the other over a the past few months.  Your advise will help me from wasting any more of my time. Regards

**Comment by semiautomaticgit** [comment]
Hello, it is an installation issue related to Python and multiprocessing. 
You can try to follow this guide https://semiautomaticclassificationmanual.readthedocs.io/en/latest/faq.html#i-have-installation-issues-on-mac-os-how-can-i-install-scp-on-mac-os .
Alternatively you can use a virtual machine.
